syncopy/datatype/methods/selectdata.py

```diff
@@ -302,9 +305,20 @@ def selectdata(data, trials=None, channels=None, channels_i=None, channels_j=Non
 
     # If an in-place selection was requested we're done
     if inplace:
-        SPYInfo("In-place selection attached to data object: {}".format(data._selection))
         return
 
+    # Inform the user what's about to happen
+    fauxTrials = [data._preview_trial(trlno) for trlno in data._selection.trials]
+    fauxSizes = [np.prod(ftrl.shape)*ftrl.dtype.itemsize for ftrl in fauxTrials]
+    selectionSize = sum(fauxSizes) / 1024**2
+    sUnit = "MB"
+    if selectionSize > 1000:
+        selectionSize /= 1024
+        sUnit = "GB"
+    msg = "Copying {dsize:3.2f} {dunit:s} of data based on selection " +\
```

Comment:
I just tried adding two AnalogData objects together, and did not receive this message, which I would have expected as this triggers an on disc copying no?!

Reply:
Hm, I see. This is actually expected since arithmetic ops do not trigger a selectdata copy: the arithmetics are `ComputationalRoutine` instances themselves, hence they perform the copying on their own. Do you want me to add a message like this `_perform_computation` as well?

Reply:
I think this would be good no? So ppl see what happens if they start chugging data around.. as even a simply `my_adata / 3` can trigger GBs of I/O operations

Reply:
Included in latest commit

Reply:
wow, even the operator appears in the new Info message, really cool :+1: